lib/keys.js

```diff
@@ -41,7 +41,20 @@ Keys.prototype.create = function(fn) {
 
     data[key] = true;
     keys.writeFile(data, function(err) {
-      fn(err, key);
+      if(err){
+        if(err.code == 'ENOENT' && err.path == '.dpd/keys.json'){
+          console.error('Path: '+err.path+' not found, creating new one.');
+          // TODO: Could ask with prompt?
+          fs.mkdirSync('.dpd');
+          keys.writeFile(data,function(error){
```

Comment:
How can we reproduce this bug (missing of '.dpd/keys.json') ?

Personally, don't find any problem for that